internal/app/machined/pkg/system/services/etcd.go

```diff
@@ -49,6 +49,10 @@ func (e *Etcd) ID(config config.Configurator) string {
 
 // PreFunc implements the Service interface.
 func (e *Etcd) PreFunc(ctx context.Context, config config.Configurator) (err error) {
+	if err = os.MkdirAll(constants.EtcdDataPath, 0755); err != nil {
+		return err
+	}
```

Comment:
not related to this change, but `fmt.Errorf` format should be `%w` for error wrapping in Go 1.13 (on line 66:70 below)

Reply:
👍 yeah I'd like to move to new error wrapping in 1.13 and remove `github.com/pkg/errors`.